environment.yml

```diff
@@ -8,11 +8,11 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python>=3.6
+  - python>=3.6,<3.10
   - asf_search
   - cartopy
-  - gdal>=3.5.1
-  - libgdal>=3.5.1
+  - gdal<=3.4.2
```

Comment:
Should this be `gdal>=3.4.2`

Reply:
We resolved this in the latest patch

Reply:
@yunjunz actually no, because we want to have a consistent environment with ISCE, which at my last check (a couple days ago) was restricted to gdal=3.4.2

Reply:
`gdal` version 3.4.2 is used by mamba from my installation as well.